test/abort.js

```diff
@@ -0,0 +1,57 @@
+var supertest = require('supertest');
+var request = require('request');
+var proxy = require('..');
+var setup = require('./setup');
+
+describe('request abortion', function() {
```

Comment:
Could you also include a unit test where supertest makes a `POST` request? That will cover the code path above where `is.hasBody(req) === true`.